engine/src/main/java/com/datatorrent/stram/client/StramAgent.java

```diff
@@ -420,8 +419,9 @@ private StramWebServicesInfo retrieveWebServicesInfo(String appId)
       JSONObject permissionsInfo = null;
       try (FSDataInputStream is = fileSystem.open(new Path(appPath, "permissions.json"))) {
         permissionsInfo = new JSONObject(IOUtils.toString(is));
-      } catch (FileNotFoundException ex) {
+      } catch (IOException ex) {
         // ignore if file is not found
+        LOG.info("Exception in accessing permissions.json", ex);
```

Comment:
The comment says "ignore". If it is important enough to WARN then I would expect a more elaborate comment.